torch_xla/core/xla_model.py

```diff
@@ -26,6 +27,22 @@
 _DEVICE_CONTEXTS = dict()
 _DEVICE_CONTEXTS_LOCK = threading.Lock()
 
+_WORLD_SIZE = None
```

Comment:
Please add a comment here to state that we cached these value for dynamo, or it might get removed one day when people don't know why we did it.

Reply:
At least they delete the test case as well. Otherwise, there won't be silent failures...

But agree, let me add a comment.